tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -509,6 +509,9 @@ public static class Tags
         internal const string GrpcMethodName = "grpc.method.name";
         internal const string GrpcStatusCode = "grpc.status.code";
 
+        internal const string ProcesTruncated = "cmd.truncated";
```

Comment:
Truncated has a different meaning here: https://datadoghq.atlassian.net/wiki/spaces/APM/pages/2357395856/Span+attributes#%5BinlineExtension%5DSystem

Because I guess the plan is to have a single integration, I think we should talk to other tracers on this.

Reply:
We are not processing the command line arguments right now (we left that out), but we are processing the environment variables. It makes sense to truncate the env variables line if it's too big and the documentation talks about truncating the command line with the arguments and the variables. Since we are not processing the arguments, only variables are left. This tag is useful to let the user know that part of the information was truncated. What do you think?

Reply:
The specifications doc says `cmd.truncated` should only be set if `cmd.exec` or `cmd.shell` was truncated. Where are we including the environment variables? In another tag?

Reply:
Ahh, I see `cmd.environment_variables`, but that is not in the specs (either the google doc RFC or the confluence page). I think it's fine to add this tag but we should probably not change the expected meaning of `cmd.truncated`.

Reply:
I think that we can skip the cmd.truncated tag, for now, since we are not processing the full command line with the arguments. Later, when we do that, this tag can be included. I will change that.